Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -45,7 +45,7 @@ class InstructionPresenter {
     
     let onShieldDownload: ShieldDownloadCompletion?
 
-    private let spriteRepository: SpriteRepository
+    let spriteRepository: SpriteRepository
```

Comment:
When the `InstructionLabel.showHighlightedTextColor` property changes, it will call the `InstructionLabel.updateLabelAttributedText()` to generate the new attributed text as well the new exit views and generic route shields inside attributed text.

So my idea is that, we should override the `InstructionLabel.showHighlightedTextColor` with a setter. When it changes, we pass the `showHighlightedTextColor` as a boolean value to the `InstructionPresenter`. Then when we generate exit views and generic route shields, we know whether they're highlighted or not. 

If `InstructionPresenter` knows it's highlighted, the exit views and generic route shields could update both `foregroundColor` and `borderColor` during generation, we could add something like a customized `highlightedColor` to its properties and apply to them. And add the highlighted boolean value to its critical hash as well.

So in this, the chain is from `InstructionLabel.showHighlightedTextColor`  to `InstructionPresenter.isHighlighted` and then to the `foregroundColor` and `borderColor` to both generic route shields and exit views. We'd better leave other components along. And it doesn't matter with the `SpriteRepository` or the `InstructionsCardView`.